modin/pandas/dataframe.py

```diff
@@ -1685,32 +1624,60 @@ def squeeze(self, axis=None):
         else:
             return self.copy()
 
-    def std(
-        self, axis=None, skipna=None, level=None, ddof=1, numeric_only=None, **kwargs
+    def _stat_operation(
+        self,
+        op_name: str,
+        axis: int,
+        skipna: bool,
+        level: Optional[Union[int, str]],
+        numeric_only: Optional[bool] = None,
+        **kwargs,
     ):
+        """
+        Do common statistic reduce operations under frame.
+
+        Parameters
+        ----------
+            op_name : str,
+                Name of method to apply.
+            axis : int,
+                Axis to apply method on.
+            skipna : bool,
+                Exclude NA/null values when computing the result.
+            level : int or level name,
+                If specified `axis` is a MultiIndex, applying method along a particular
+                level, collapsing into a Series
+            numeric_only : bool
+                Include only float, int, boolean columns. If None, will attempt
+                to use everything, then use only numeric data.
```

Comment:
```suggestion
        op_name: str,
            Name of method to apply.
        axis: int,
            Axis to apply method on.
        skipna: bool,
            Exclude NA/null values when computing the result.
        level: int or level name,
            If specified `axis` is a MultiIndex, applying method along a particular
            level, collapsing into a Series
        numeric_only: bool
            Include only float, int, boolean columns. If None, will attempt
            to use everything, then use only numeric data.
```

Reply:
@anmyachev , I know we are using style `parameter name: parameter type` since some point. However, I would prefer using `parameter name : parameter type`. I am not sure but [mentioned extension in sphinx](https://github.com/modin-project/modin/issues/1411#issuecomment-622065970) may not work correctly with this. I think we need to avoid any inconsistencies with numpydoc.

Reply:
@YarShev I don't mind using either of these two styles. Let's discuss this elsewhere.